cmd/tk/workflow.go

```diff
@@ -137,6 +137,16 @@ func showCmd() *cobra.Command {
 	}
 	vars := workflowFlags(cmd.Flags())
 	cmd.Run = func(cmd *cobra.Command, args []string) {
+		fi, err := os.Stdout.Stat()
+		if err != nil {
+			log.Fatalln("stdout pipe detection:", err)
```

Comment:
This is a safe-guard feature and should not make the whole process crash.

If we cannot tell what happens, just optimistically print the results